src/pickley/__init__.py

```diff
@@ -117,7 +117,7 @@ def update_from_pip_show(self, venv, canonical_name):
         self.canonical_name = canonical_name
         self.values = {}
         r = venv.run_pip("show", canonical_name, fatal=False)
-        if r.failed:
+        if r.failed:  # pragma: no cover, hard to trigger without excessive mocking
```

Comment:
Is this really "excessive"?  My preference would be to leave paths that users could actually hit uncovered to remind me to cover them someday :)

```
def test():
  result = Mock(failed=True, full_output="foo")
  venv = Mock(run_pip=Mock(side_effect=result))
  pm = PipMetadata()
  pm.update_from_pip_show(venv, "cn")
  assert pm.problem == "Failed to `pip show cn`: foo"
```